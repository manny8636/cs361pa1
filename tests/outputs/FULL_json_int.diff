--- outputs/FULL_json_int.txt	2022-02-15 14:30:56.683086554 -0500
+++ expected/FULL_json_int.txt	2021-01-25 15:27:24.000000000 -0500
@@ -0,0 +1,5 @@
+Key-value pairs:
+{
+KEYS[keyName] = 125
+}
+Success!
