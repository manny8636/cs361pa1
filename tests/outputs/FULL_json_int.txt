Parsing ../data/json_int.txt failed
