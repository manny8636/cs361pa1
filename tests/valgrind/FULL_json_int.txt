==372176== Memcheck, a memory error detector
==372176== Copyright (C) 2002-2017, and GNU GPL'd, by Julian Seward et al.
==372176== Using Valgrind-3.15.0 and LibVEX; rerun with -h for copyright info
==372176== Command: ../json -o ../data/json_int.txt
==372176== 
==372176== 
==372176== HEAP SUMMARY:
==372176==     in use at exit: 2,064 bytes in 1 blocks
==372176==   total heap usage: 4 allocs, 3 frees, 10,848 bytes allocated
==372176== 
==372176== LEAK SUMMARY:
==372176==    definitely lost: 2,064 bytes in 1 blocks
==372176==    indirectly lost: 0 bytes in 0 blocks
==372176==      possibly lost: 0 bytes in 0 blocks
==372176==    still reachable: 0 bytes in 0 blocks
==372176==         suppressed: 0 bytes in 0 blocks
==372176== Rerun with --leak-check=full to see details of leaked memory
==372176== 
==372176== For lists of detected and suppressed errors, rerun with: -s
==372176== ERROR SUMMARY: 0 errors from 0 contexts (suppressed: 0 from 0)
