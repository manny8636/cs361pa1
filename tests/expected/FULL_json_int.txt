Key-value pairs:
{
KEYS[keyName] = 125
}
Success!
